frontend/public/components/sidebars/explore-type-sidebar.tsx

```diff
@@ -49,8 +52,9 @@ export const ExploreType: React.FC<ExploreTypeProps> = (props) => {
     ? currentSelection.description
     : currentDefinition.description;
   const required = new Set(currentDefinition.required || []);
+  const kindLabel = kindObj.labelKey ? t(kindObj.labelKey) : kindObj.kind;
   const breadcrumbs = drilldownHistory.length
-    ? [kindObj ? kindObj.kind : 'Schema', ..._.map(drilldownHistory, 'name')]
+    ? [kindObj ? kindLabel : t('public~Schema'), ..._.map(drilldownHistory, 'name')]
```

Comment:
We check if `kindObj` is set above (line 31), so I don't think this fallback is needed.

Reply:
You can have no kindObj as long as you have a schema, so you do need the fallback.